### DISCUSSION TITLE
Hbase的支持

### DISCUSSION THREAD

**Discussion by yqMac** [hhyo/Archery#2870] [Ideas]
虽然在这个里面说不太合理，不过我们生产确实用的archery ，Hbase的查询和插入、更新数据，我很好奇这块有没有规划能支持。

**Comment by LeoQuote** [comment]
可以参考下这里： https://github.com/hhyo/Archery/wiki/development#%E6%95%B0%E6%8D%AE%E5%BA%93%E5%BC%95%E6%93%8E
